lib/app_manifest.rb

```diff
@@ -36,9 +36,9 @@ def canonicalize_env(manifest)
       canonicalize_key(manifest, :env) do |env|
         Hash[
           env.map do |key, value|
-            if value.is_a? String
+            if !value.is_a?(Hash)
               value = {
-                value: value,
+                value: value.to_s,
```

Comment:
+1 on what @gudmundur said. I figured this change in app-manifest was motivated by a support ticket that came in last week about using boolean values for envs. I assumed this change was already in place in Nyata, but until we don't accept that in app-setups, I don't think we should make the change in this gem.